<launch>
    <node name="lidar_fusion_node" pkg="lidar_fusion" type="lidar_fusion_node" output="screen">
    <param name="config_path" value="$(find lidar_fusion)/config/config.yaml" />
</node>
</launch>